doc/aaa/SONiC RBAC v2.md

```diff
@@ -281,11 +308,10 @@ Users and their role (group) assignments may be managed via Linux tools or the N
 
 * **PrivilegeTable**
 
-  This table has provides the information about the type of operations that a particular role is authorized to perform. The authorization can be performed at the granularity of a feature, feature group, or the entire system. The table has the following columns :
+  This table provides the information about the type of operations that a particular role is authorized to perform. The authorization can be performed at the granularity of a feature, feature group, or the entire system. The table has the following columns :
     * *role* : The role associated with the user that is being authorized. This is a string.
     * *feature* : This is feature that the role is being authorized to access. The granularity of the feature can be :
-        * *feature* - A logical grouping of multiple commands. If the user is authorized to access a particular feature, the column contains the tag associated with that feature. (More on tagging later. This will be implemented in Phase 2 of RBAC.)
-        * *feature-group* - A logical grouping of multiple features. If the user is authorized to a feature-group, the column contains the name of the feature-group. (More on feature-group later. This will be implemented in Phase 2 of RBAC.)
+        * *feature* - A logical grouping of multiple commands. If the user is authorized to access a particular feature, the column contains the tag associated with that feature.
```

Comment:
Maybe this should be called something other than "feature". This actually sounds like "feature-group". And then give some examples of what this logical grouping might be.